src/app/containers/omsorgspenger/korrigeringAvInntektsmelding/KorrigeringAvInntektsmeldingForm.tsx

```diff
@@ -44,15 +47,44 @@ const initialFormState = {
     innsendteFormverdier: undefined,
 };
 
+const getPeriodRange = (fom: string, tom: string) => {
+    const dager = [];
+    let currentDate = initializeDate(fom);
+    dager.push(currentDate.format('YYYY-MM-DD'));
+    while (currentDate.isBefore(tom) || currentDate.isSame(tom)) {
```

Comment:
mulig jeg tenker feil, men vil ikke denne gi deg en dag for mye? Den legger til currentDate + 1 i dager når currentDate er det samme som TOM

Reply:
Tror du tenker rett. Det fungerer som det er nå, men isSame gir false når datoene er like av en eller annen grunn. Skal fikse